docs/src/main/asciidoc/_app-dev-guide/assemblies/assembly_deploying-an-application-to-the-server.adoc

```diff
@@ -0,0 +1,36 @@
+ifdef::context[:parent-context-of-deploying-an-application-to-the-server: {context}]
+
+:_content-type: ASSEMBLY
+
+ifndef::context[]
+[id="deploying-an-application-to-the-server"]
+endif::[]
+ifdef::context[]
+[id="deploying-an-application-to-the-server_{context}"]
+endif::[]
+= Deploying an application to the server
+
+:context: deploying-an-application-to-the-server
+
+[role="_abstract"]
+You can deploy your application on a {ProductShortName} server running on bare metal or on {cloudPlatform}.
+
+To deploy your application on a {ProductShortName} server running on bare metal, follow this procedure:
+
+* xref:deploying-an-application-to-a-bare-metal-installation_deploying-an-application-to-the-server[Deploying an application to a bare metal installation]
+
+To deploy your application on a {ProductShortName} server running on {cloudPlatform}, follow these procedures:
+
+* xref:preparing-an-application-for-deployment-on-openshift_deploying-an-application-to-cloudplatform[Preparing an application for deployment on {cloudPlatform}]
+* xref:deploying-an-application-to-server-on-openshift-with-helm_deploying-an-application-to-cloudplatform[Deploying an application to {ProductShortname} on OpenShift with Helm]
+
+include::{pathToModules}/proc_deploying-an-application-to-a-bare-metal-installation.adoc[leveloffset=+1]
+
```

Comment:
We need the server provisioning section here, which is upstream only content

Reply:
added